xrootd/client/lookupgid_windows.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build windows
+
+package client // import "go-hep.org/x/hep/xrootd/client"
+
+import (
+	"os/user"
+)
+
+func lookupGroupID(usr *user.User) (string, error) {
+	return usr.Username, nil
```

Comment:
perhaps add some comment here?
because for somebody not super familiar with Windows (like me), it looks like a bug or a typo :)